runtime-config.md

```diff
@@ -36,6 +36,15 @@ Only [mounts from the portable config](config.md#mount-points) will be mounted.
     }
 }
 ```
+Note: The following filesystems MUST be made available in each Linux application's filesystem
```

Comment:
On Thu, Sep 10, 2015 at 08:46:15PM -0700, Gao feng wrote:
> > +Note: The following filesystems MUST be made available in each Linux application's filesystem
> 
> @wking I thinks describe this after the mounts Exmaple for linux…

That's a good point.  I guess my preferred approach would be to move
the Linux-specific mount example into the Linux-specific
documentation, and just link to the per-platform examples from this
platform-agnostic documentation.  Otherwise this is a reasonable chunk
of noise for folks working on other platforms.